.github/workflows/ci.yml

```
@@ -54,10 +54,10 @@ jobs:
     name: Documentation
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - uses: julia-actions/setup-julia@latest
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
         with:
-          version: '1.4'
+          version: '1'
```

```suggestion
          version: '1.6'
```

(just so you can press it in the online interface; then I think we could merge this immediately after)